tests/tasks/test_dao_complete_task.py

```diff
@@ -20,7 +20,7 @@ def test_achieve_not_achieved_task(self):
     def test_achieve_achieved_task(self):
         self.assertTrue(self.tasks_list_1.find_task_by_id(2).get("is_done"))
 
-        expected_response = messages.TASK_WAS_ALREADY_ACHIEVED, 403
+        expected_response = messages.TASK_WAS_ALREADY_ACHIEVED, 409
```

Comment:
Hi @divyanshugit please use `HTTPStatus.CONFLICT.value` here as suggested 
```suggestion
        expected_response = messages.TASK_WAS_ALREADY_ACHIEVED, HTTPStatus.CONFLICT.value
```